extensions/dwdm/dwdm.yml

```diff
@@ -2,18 +2,220 @@
+          - name: LSH_APC
+            label: LSH/APC
+            description: "Angled Physical Contact version of LSH, designed to reduce back reflections."
+          - name: LX_5
+            label: LX.5
+            description: "Miniaturized fiber optic connector similar to LC but with an additional shutter mechanism."
+          - name: LX_5_PC
+            label: LX.5/PC
+            description: "Physical Contact version of LX.5."
+          - name: LX_5_UPC
+            label: LX.5/UPC
+            description: "Ultra-Physical Contact variant of LX.5."
+          - name: LX_5_APC
+            label: LX.5/APC
+            description: "Angled Physical Contact version of LX.5."
+          - name: MPO
+            label: MPO
+            description: "Multi-fiber Push-On connector typically used in data centers for high-speed applications."
+          - name: MTRJ
+            label: MTRJ
+            description: "Male-to-female fiber optic connector with two fibers."
+          - name: SC
+            label: SC
+            description: "Square fiber optic connector with push-pull lock."
+          - name: SC_PC
+            label: SC/PC
+            description: "Physical Contact SC connector with a polished end-face."
+          - name: SC_UPC
+            label: SC/UPC
+            description: "Ultra-Physical Contact variant of SC."
+          - name: SC_APC
+            label: SC/APC
+            description: "Angled Physical Contact version of SC."
+          - name: ST
+            label: ST
+            description: "Bayonet-style fiber optic connector primarily used in industrial and military applications."
+          - name: CS
+            label: CS
+            description: "Compact connector with a high-density duplex configuration."
+          - name: SN
+            label: SN
+            description: "Small-form connector with dual fibers."
+          - name: SMA_905
+            label: SMA 905
+            description: "Stainless steel fiber optic connector."
+          - name: SMA_906
+            label: SMA 906
+            description: "Variant of SMA 905 with similar durability, frequently used in high-vibration settings."
+          - name: URM_P2
+            label: URM-P2
+            description: "Specialized fiber optic connector for industrial and harsh environments."
+          # FIXME: Commenting out otherwise I'm facing a Text attribute length should be less than 4096 characters.
+          # - name: URM_P4
+          #   label: URM-P4
+          #   description: "Similar to URM-P2 but designed for higher performance."
+          # - name: URM_P8
+          #   label: URM-P8
+          #   description: "Enhanced version of URM connectors with higher protection."
+          # - name: Splice
+          #   label: Splice
+          #   description: "Permanent fiber connection method where two fiber ends are fused."
+
 nodes:
-  - name: DWDMChannel
+  - name: DwdmDevice
```

Comment:
Should we just call it a MUX / DEMUX ?

Reply:
OADM (Optical Add Drop Multiplexer) is apparently the exact name ...

Should we call that "Optical Multiplexer" (rip demux) and put the entire name in the description or something?

Reply:
yep let's do that